queries.sql

```diff
@@ -43,3 +43,24 @@ SELECT neutered, COUNT(*) AS escape_count  FROM animals WHERE escape_attempts >
  SELECT species , MIN(weight_kg) AS min_weight , MAX(weight_kg) AS  max_weight FROM animals GROUP BY species;
  SELECT species , MIN(weight_kg) AS min_weight , MAX(weight_kg) AS  max_weight FROM animals GROUP BY species;
  SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+SELECT a.name FROM animals a JOIN species s ON a.species_id = s.id WHERE s.name = 'Pokemon';
+SELECT o.full_name , a.name AS animal_name FROM owners o JOIN animals a ON a.owner_id = o.id;
+SELECT s.name, COUNT(a.id) AS animal_count FROM species s LEFT JOIN animals a ON s.id = a.species_id GROUP BY s.name;
+SELECT o.full_name ,COUNT(a.id) AS animal_count FROM owners o LEFT JOIN animals a ON a.owner_id = o.id GROUP BY o.full_name ORDER BY animal_count DESC LIMIT 1;
+SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id WHERE o.full_name = ' Dean Winchester' AND a.escape_attempts = 0;
+SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id JOIN species s  ON a.species_id = s.id WHERE  o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
+
+
+
+SELECT vt.name  AS vet_name from vets vt LEFT JOIN specializations sp ON vt.id = sp.vet_id LEFT JOIN species s ON sp.species_id = s.id;
+SELECT vt.name  AS vet_name, s.species AS Specialty_name from vets vt LEFT JOIN specializations sp ON vt.id = sp.vet_id LEFT JOIN species s ON sp.species_id = s.id;
+SELECT a.name AS animal_name FROM animals a JOIN  visits v ON a.id = v.animal_id JOIN  vets vt ON v.vet_id = vt.id WHERE vt.name = 'Vet Stephanie Mendez' AND v.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
+SELECT a.name AS animal_name , COUNT(v.animal_id) AS visit_count from animals a JOIN visits v ON v.animal_id = a.id  GROUP BY a.name  ORDER BY visit_count DESC LIMIT 1;
+SELECT vt.name AS Vet_name FROM visits v JOIN vets vt ON v.vet_id = vt.id JOIN animals a ON v.animal_id = a.id WHERE a.name = 'Vet Maisy Smith' ORDER BY v.visit_date ASC LIMIT 1;
+SELECT a.name AS animal_name  FROM visits v JOIN vets vt ON v.vet_id = vt.id JOIN animals a ON v.animal_id = a.id WHERE a.name = 'Vet Maisy Smith' ORDER BY v.visit_date ASC LIMIT 1;
+SELECT vt.name AS Vet_name FROM visits v JOIN vets vt ON v.vet_id = vt.id JOIN animals a ON v.animal_id = a.id WHERE a.name = 'Vet Maisy Smith' ORDER BY v.visit_date ASC LIMIT 1;
+SELECT a.name  AS animal_name , vt.name AS vet_name, v.visit_date FROM visits v JOIN vets vt ON v.vet_id = vt.id JOIN animals a ON v.animal_id = a.id ORDER BY v.visit_date  DESC LIMIT 1;
+SELECT COUNT(*) AS "Visits Without Specialization"FROM visits v JOIN animals a ON v.animal_id = a.id JOIN vets vet ON v.vet_id = vet.id LEFT JOIN specializations s ON vet.id = s.vet_id AND a.species = (SELECT species FROM species WHERE id = s.species_id) WHERE s.vet_id IS NULL;
```

Comment:
- You have done well so far, but the attached answer in your PR, for this query, `How many times visits where with a vet that did not specialize in that animal's species?`, is wrong. Please re-run your query and attach the correct answer in your PR. 
``` SQL
       SELECT count(*)
      FROM visits
      LEFT JOIN animals ON animals.id = visits.animal_id
      LEFT JOIN vets ON vets.id = visits.vet_id
      WHERE animals.species_id NOT IN (SELECT species_id FROM specializations WHERE vet_id = vets.id)
```